xapian-letor/include/xapian-letor/ranker.h

```diff
@@ -381,37 +381,37 @@ class XAPIAN_VISIBILITY_DEFAULT ListMLERanker: public Ranker {
      *
```

Comment:
This isn't how the model is saved - we actually use `Xapian::serialise_double()` (which gives a much more compact result, and can round-trip values exactly, which is harder to reliably achieve by writing out values as text strings because the same real values aren't exactly representable in base 2 and base 10).

I don't think this is the right place to document exactly how the model is serialised though (and such documentation probably needs to go into more detail than just telling us it's a list of parameters).  API users don't need to worry about how the model is serialised, because they shouldn't need to try to unpick the data - they'll use the xapian-letor API to unserialise it.

Same issue with the doc comment for the same method of `ListNetRanker`.